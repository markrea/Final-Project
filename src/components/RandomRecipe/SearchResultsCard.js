import React from "react";
import Proptypes from "prop-types";
import SearchResults from "./SearchResults";
import "../../styles/App.css";

const SearchResultsCard = ({ searchResults }) => {
  return (
    <div>
      <div className="search-results" data-testid="search-div">
        {searchResults && <SearchResults searchResults={searchResults} />}
      </div>
    </div>
  );
};
SearchResultsCard.propTypes = {
  searchResults: Proptypes.objectOf.isRequired,
};

export default SearchResultsCard;
