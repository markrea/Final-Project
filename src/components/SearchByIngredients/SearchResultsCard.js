import React from "react";
import SearchResults from "./SearchResults";
import "../../styles/App.css";

const SearchResultsCard = ({searchResults}) => {
  return (
    <div>
      <div className="search-results" data-testid="search-div">
        {searchResults ? (
          <SearchResults searchResults={searchResults} />
        ) : (
          <div>
            <h2 className="results-text"></h2>
          </div>
        )}
      </div>
    </div>
  );
};
export default SearchResultsCard;
