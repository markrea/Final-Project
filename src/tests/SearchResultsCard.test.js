import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import SearchResultsCard from "../components/SearchResultsCard";
import "@testing-library/jest-dom/extend-expect";

describe("SearchResultsCard", () => {
  it("component renders correctly", () => {
    const { asFragment } = render(<SearchResultsCard />);
    expect(asFragment).toMatchSnapshot();
  });
});
