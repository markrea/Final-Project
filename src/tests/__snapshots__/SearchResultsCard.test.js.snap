// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResultsCard component renders correctly 1`] = `[Function]`;
